src/current/cockroachcloud/frequently-asked-questions.md

```diff
@@ -1,202 +0,0 @@
----
```

Comment:
I don't know the answer to this, but do we want to delete the file totally? Or, just remove from the sidebar — you / Matt pointed out some reasons why this may have existed and so I wasn't sure if we wanted to do a "test" removal first. I'm sure most of this content is captured elsewhere in the docs, however that's my only pause that we might be accidentally removing content from its only place in the docs.

Reply:
I'm pretty sure most of the content is captured/will be captured elsewhere (the uncaptured content will go to the _Plan a cluster_ docs when I get to them). I am in favor of deleting the files since this feels like a good opportunity to pare down our docs set. If we don't delete these files now, they'll linger until the next big overhaul 😬